reporter/html.js

```diff
@@ -676,20 +676,20 @@ QUnit.log( function( details ) {
 	// Also, we need to check details.expected existence, as it can exist and be undefined
 	if ( !details.result && hasOwn.call( details, "expected" ) ) {
 		if ( details.negative ) {
-			expected = escapeText( "NOT " + QUnit.dump.parse( details.expected ) );
+			expected = "NOT " + QUnit.dump.parse( details.expected );
 		} else {
-			expected = escapeText( QUnit.dump.parse( details.expected ) );
+			expected = QUnit.dump.parse( details.expected );
 		}
 
-		actual = escapeText( QUnit.dump.parse( details.actual ) );
+		actual = QUnit.dump.parse( details.actual );
```

Comment:
I would like to see assertions covering the behavior of `QUnit.diff` [when provided strings containing escaped and/or unescaped HTML](https://jsfiddle.net/gL6djqm0/) (and also Javascript escape sequences, for that matter). Looking at [test/reporter-html/diff.js](https://github.com/jquery/qunit/blob/0f9ae3443a0fc78d96baa2eb86f90b0cba9264b5/test/reporter-html/diff.js), I see this as a sizable gap.